packages/director/src/app.ts

```diff
@@ -101,12 +115,18 @@ router.post(
   catchRequestHandlerErrors(async (req, res) => {
     const executionDriver = await getExecutionDriver();
     if (executionDriver.id !== 'in-memory')
-      return res.status(405).send('This is only available for in-memory db. Please use the dashboard to set your hooks.');
+      return res
+        .status(405)
+        .send(
+          'This is only available for in-memory db. Please use the dashboard to set your hooks.'
+        );
 
     const { projectId, hooks } = req.body;
     executionDriver.setHooks && executionDriver.setHooks(projectId, hooks);
     getLogger().log(`[hooks] Hooks set for project ${req.body.projectId}`);
-    return res.status(200).send(`Hooks set for project "${req.body.projectId}".`);
+    return res
```

Comment:
This is only a change because the settings from prettierrc.js were applied correctly.